RevenueCatUI/Data/Constants.swift

```diff
@@ -74,4 +74,21 @@ extension TemplateViewType {
         return Constants.defaultVerticalPaddingLength(self.userInterfaceIdiom)
     }
 
+    var isVerticalSizeCompact: Bool {
+        #if os(tvOS)
+        // tvOS never reports UserInterfaceSizeClass.compact
+        // but for the purposes of template layouts, we consider landscape
+        // on tvOS as compact to produce horizontal layouts.
```

Comment:
this is a fair point, but it makes me wonder whether the flag should be named something more representative, like `shouldUseLandscapeLayout` or something, given that it's not really "compact" for tvOS and that on macOS it's neither compact nor not compact, right?